SplunkPy- merge pre-release in to the official (#16576)
demisto/content: Packs/SplunkPy/ReleaseNotes/2_2_10.md

#### Integrations
##### SplunkPy

> - This integration version is now aligned to the **SplunkPyPreRelease** version 1.0.4 which introduced significant change on the fetch mechanism.

Reviewer: ```suggestion
- This integration version is now aligned to version 1.0.4 of the **SplunkPyPreRelease** pack, which introduced a major change to the *fetch incidents* mechanism.
```
